packages/textinput/src/react/index.js

```diff
@@ -93,6 +99,7 @@ TextInput.propTypes = {
   appearance: PropTypes.oneOf(Object.keys(vars.appearances)),
   disabled: PropTypes.bool,
   error: PropTypes.bool,
+  fieldAfter: PropTypes.node,
```

Comment:
What was your eventual thought on putting  an underscore before the name, ie `_fieldAfter`, in order to discourage use?  

Or perhaps just not documenting it is enough?

Reply:
Decided:
- No doc
- No under, that means inter-compound component props
- Might use later and expose as more general extensibility.